app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy: int = 0
+    not_vaccinated: bool = False
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            not_vaccinated = True
```

Comment:
Instead of returning immediately when a `VaccineError` is caught, set a flag to indicate that at least one friend is not vaccinated. This will allow the loop to continue checking other friends before returning a final message.